third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1339,18 +1339,20 @@ func TestAccContainerCluster_withIPAllocationPolicy_createSubnetwork(t *testing.
 				Config: testAccContainerCluster_withIPAllocationPolicy_createSubnetwork(cluster),
 			},
 			{
-				ResourceName:		 "google_container_cluster.with_ip_allocation_policy",
+				ResourceName:		     "google_container_cluster.with_ip_allocation_policy",
 				ImportStateIdPrefix: "us-central1-a/",
-				ImportState:		 true,
+				ImportState:		     true,
 				ImportStateVerify:	 true,
 			},
 			{
 				Config: testAccContainerCluster_withIPAllocationPolicy_createSubnetworkUpdated(cluster),
+				PlanOnly:           true,
+				ExpectNonEmptyPlan: false,
 			},
 			{
-				ResourceName:		 "google_container_cluster.with_ip_allocation_policy",
+				ResourceName:		     "google_container_cluster.with_ip_allocation_policy",
```

Comment:
do we still want this import step given that the previous was planonly?

Reply:
🤷‍♀ it's probably entirely redundant, but import is free and running ISV another time is nice.